### PR TITLE
Scroll & Animation fix

### PR DESCRIPTION
**Pull request by SachinRammoorthy**
Fixes include: 
- Scrolling issues,
- Minor issue in animation,
- Added GCI logo & linked to JBoss GCI page.

Solves issue #46

### CODE REVIEW

**Comment by SachinRammoorthy** [comment]
@aashutoshrathi Added more changes and squashed commits. Please have a look.

**Comment by aashutoshrathi** [comment]
Something wrong with students again, as contributors starting to load.

**Comment by SachinRammoorthy** [comment]
Yes that is a problem that I am stuck on. The issue is that the contributors section loads only after 3-4 seconds, and so the page scrolls to the initial placement of the students section. Will try to come up with a fix now.

**Comment by SachinRammoorthy** [comment]
@aashutoshrathi have come up with a primitive solution 😅 . Please let me know if its alright.

**Comment by aashutoshrathi** [comment]
Nice try @SachinRammoorthy, but seriously we don't fix a bug like that, please try to look deeper into the issue. 😅

File: index.html [hunk 1]
```diff
@@ -143,6 +145,19 @@ <h2>Contributors</h2>
                                                       
           <!-- End of Contributers Section -->
 
+          <!-- Blank Space (for Contributor section -->
+          <section>
+            <br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br>
```
Comment (aashutoshrathi) [inline index.html]: Can you give it a try?
```
<br style = “line-height:x;”><br>
Replace “x” with the number of lines you want blank.
```

File: index.html [hunk 2]
```diff
@@ -38,18 +38,19 @@
...
-              <li><a class="menu-item" href="#issues">Issues</a></li>
-              <li><a class="menu-item" href="https://blog.jboss-outreach.org/">Blog</a></li>
+              <li><a class="menu-item" href="#welcome"><h3>Welcome</h3></a></li>
+              <li><a class="menu-item" href="#mentors"><h3>Mentors</h3></a></li>
+              <li><a class="menu-item" href="#contributors"><h3>Contributors</h3></a></li>
+              <li><a class="menu-item" href="#students"><h3>Students</h3></a></li>
+              <li><a class="menu-item" href="#issues"><h3>Issues</h3></a></li>
+              <li><a class="menu-item" href="https://blog.jboss-outreach.org/"><h3>Blog</h3></a></li>
               </ul>
+              <img src="./images/gci_logo.png" height="75" style="margin-top: 10px; margin-left: 35px;">
```
Comment (aashutoshrathi) [inline index.html]: Also better if you link this image to https://codein.withgoogle.com/organizations/jboss-community/

**Comment by SachinRammoorthy** [comment]
@aashutoshrathi have worked on the changes.

File: index.html [hunk 3]
```diff
@@ -38,18 +38,21 @@
...
+              <li><a class="menu-item" href="#welcome"><h3>Welcome</h3></a></li>
+              <li><a class="menu-item" href="#mentors"><h3>Mentors</h3></a></li>
+              <li><a class="menu-item" href="#contributors"><h3>Contributors</h3></a></li>
+              <li><a class="menu-item" href="#students"><h3>Students</h3></a></li>
+              <li><a class="menu-item" href="#issues"><h3>Issues</h3></a></li>
+              <li><a class="menu-item" href="https://blog.jboss-outreach.org/"><h3>Blog</h3></a></li>
               </ul>
+              <a href="https://codein.withgoogle.com/organizations/jboss-community/">
+              <img src="./images/gci_logo.png" height="75" style="margin-top: 10px; margin-left: 35px;">
+            </a>
```
Comment (aashutoshrathi) [inline index.html]: Fix the indentation of this, and don't squash this time so that I can check the demo link. 😄

Comment (SachinRammoorthy) [reply index.html]: Fixed indentation.

**Comment by aashutoshrathi** [comment]
Squash now, nice work looks good to me 😄

**Comment by SachinRammoorthy** [comment]
@aashutoshrathi commits have been squashed

**Comment by aashutoshrathi** [comment]
Ahh.. @SachinRammoorthy I just noticed you always send PR from the master of your fork, please don't do so and make a new branch for each PR. 😄

**Comment by SachinRammoorthy** [comment]
@aashutoshrathi how come some changes do not show up on the website?
screen shot 2017-12-27 at 6 38 34 pm
screen shot 2017-12-27 at 6 38 27 pm

**Comment by aashutoshrathi** [comment]
Clear Cache 😉

**Pull request merged**